bcdevices/ly11-system-rpi4
# PLTOS-229 Re-add pyOCD, generic CMSIS-DAP support

- Test pyocd update
- upgrade packages version
- add python wheel
- Attempt to add python wheel
- Try custom buildroot patches
- Update packages version
- Read external repository
- Use buildbot github token to check out nerves_system_br
- Remove unnecessary nerves_system_br changes
- Fix version
- Fix package extensions
- Update python packages
- Add python zipp
- Update packages version
- update packages version
- Update package version
- update package version
- Sort packages
- Fix pythib libusb package
- Fix pickle install
